src/resources/views/configuration/users/list.blade.php

```diff
@@ -8,124 +8,83 @@
-                               title="{{ trans('web::seat.delete') }}"
-                               class="confirmlink btn btn-danger">
-                              <i class="fa fa-times"></i> {{ trans('web::seat.delete') }}
-                            </a>
-                          @else
-                            <a href="{{ route('configuration.users.delete', ['user_id' => $user->id]) }}"
-                               title="{{ trans('web::seat.delete') }}"
-                               class="confirmlink disabled btn-danger btn">
-                              <i class="fa fa-times"></i> {{ trans('web::seat.delete') }}
-                            </a>
-                          @endif
-
-                        </div>
-                      </div>
-
-                    </div>
-
-                  </li>
-                @endforeach
-              </ul>
-
-            </td>
-          </tr>
-
-        @endforeach
-        </tbody>
       </table>
 
     </div>
-    <div class="panel-footer">
-      <b>{{ count($groups) }}</b> {{ trans_choice('web::seat.group', count($groups)) }} |
-      <b>{{ $groups->map(function($r) { return $r->users->count(); })->sum() }}</b> {{ trans_choice('web::seat.user', 2) }}
-    </div>
-
   </div>
 
-@stop
\ No newline at end of file
+@stop
+
+@push('javascript')
+
+  @include('web::includes.javascript.id-to-name')
+
+  <script type="text/javascript">
+    $('#user-configuration-table').DataTable({
+      "processing": true,
+      "serverSide": true,
+      "pageLength": 50,
+      "ajax": {
+        url: "{{url()->current()}}"
+      },
+      columns: [
+        {data: 'refresh_token', name: 'refresh_token', searchable: false, orderable: false},
+        {data: 'name', name: 'name'},
+        {data: 'last_login', name: 'last_login'},
+        {data: 'last_login_source', name: 'last_login_source'},
+        {data: 'action_buttons', name: 'action_buttons', searchable: false, orderable: false},
+        {data: 'roles', name: 'group.roles.title', visible: false},
+        {data: 'email', name: 'email', visible: false},
+        {data: 'main_character', name: 'main_character',visible: false}
+      ],
+      rowGroup: {
+        startRender: function(rows, group) {
+
+          var email = rows.data().pluck('email')[0];
+
+          var wraped_role_titles = wordwrap(rows.data().pluck('roles').unique().join(', '), 100, '<br/>n');
+
+          var character_group = rows.data().pluck('main_character_blade')[0];
+
+          return '{{trans('web::seat.main_character')}}: ' + character_group
+              + '{{ trans('web::seat.email') }}: ' + email
+              + '<span class="pull-right"> {{ trans_choice('web::seat.role', 2) }}: ' + wraped_role_titles + '</span>';
+        },
+        dataSrc: 'main_character'
+      },
+      drawCallback : function () {
+        $("img").unveil(100);
+        ids_to_names();
+      },
+    });
+
+    function wordwrap( str, width, brk, cut ) {
```

Comment:
I noticed the use of this function again, another candidate for `seat.js` I think. Could you maybe update the other PR's that have this too? This way we have a single origin for the code, less copy paste and have more maintainable code :D